cache/contenthash/checksum_windows.go

```diff
@@ -0,0 +1,16 @@
+package contenthash
+
+import (
+	"path/filepath"
+
+	"github.com/Microsoft/go-winio"
+)
+
+// elevating the admin privileges to walk special files/directory
```

Comment:
nit: the comments should come at L12 (just above `privileges := ..`. When placed here, they are confused with the documentation for the whole `walk` method.